submissions/andrewklmn/task_js_dom/app.js

```diff
@@ -0,0 +1,209 @@
+const state = {
+  items: [
+    {
+      menu: 'Ingénue',
```

Comment:
It's not a blocker, but I'd recommend renaming some of the properties of the objects in the `items` array for making your data easy to understanding.
```suggestion
      menuTitle: 'Ingénue',
      title: "The Ingénue",
      image: "img/beautiful-blonde.jpg",
      description: "...",
      location: "Santa Monica pier",
      attractionMethod: "..."
```